src/controllers/userController.js

```diff
@@ -48,6 +48,21 @@ const updateUser = async (req, res) => {
   });
 };
 
+const duplicateNicknameCheck = async (req, res) => {
+  const { nickname } = req.body;
+
+  keyCheck({
+    nickname,
+  });
+
+  await userService.duplicateNicknameCheck(nickname);
+
+  res.status(200).json({
+    message: "AVAILABLE_NICKNAME",
+  });
+};
+
```

Comment:
```suggestion
const checkDuplicatedNickname = async (req, res) => {
  const { nickname } = req.body;

  keyCheck({
    nickname,
  });

  await userService.checkDuplicatedNickname(nickname);

  res.status(200).json({
    message: "AVAILABLE_NICKNAME",
  });
};

```

- duplicate로 시작하는 함수 이름이 닉네임을 복제한다는 의미로 오인될 가능성이 있습니다 수정 부탁드립니다! :)

Reply:
수정했습니다!